banksystem/banks/views.py

```diff
@@ -1,5 +1,10 @@
-from django.shortcuts import render, HttpResponse
+from django.shortcuts import render
+from .models import Bank
 
 
-def index(request):
-    return HttpResponse('Welcome to the Bank')
+def get_banks(request):
+    user = request.user
+    user_banks = Bank.objects.filter(bank_accounts__user=user)
```

Comment:
study what the sql of this query actually looks like

Reply:
SELECT "banks_bank"."id", "banks_bank"."name", "banks_bank"."city" FROM "banks_bank" INNER JOIN "accounts_account" ON ("banks_bank"."id" = "accounts_account"."bank_id") WHERE "accounts_account"."user_id" = 2